inst/templates/easydashboard.Rmd

```diff
@@ -257,16 +263,16 @@ text_report_performance <- tryCatch(
 )
 
 if (!is.null(text_report)) {
-  cat(gsub("]", ")", gsub("[", "(", text_report, fixed = TRUE), fixed = TRUE))
+  bracket_to_parantheses(text_report)
```

Comment:
Not sure if you need `cat()` here, because the print is inside an if-statement.

Reply:
Good catch. We do need it!